assets/javascripts/kitten/components/menus/header-menu/index.js

```diff
@@ -1,52 +1,60 @@
-import React, { Component } from 'react'
+import React from 'react'
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
 import { Item } from './components/item'
 import { Context } from './components/context'
 import COLORS from '../../../constants/colors-config'
+import { pxToRem } from '../../../helpers/utils/typography'
+
+const borderStyle = `${pxToRem(1)} solid ${COLORS.line1}`
 
 const List = styled.ul`
   margin: 0;
   padding: 0;
   list-style: none;
-  border-left: 1px solid ${COLORS.line1};
-  border-right: 1px solid ${COLORS.line1};
+  border-left: ${({ noBorder }) => (noBorder ? 0 : borderStyle)};
+  border-right: ${({ noBorder }) => (noBorder ? 0 : borderStyle)};
 `
 
-export class HeaderMenu extends Component {
-  static Item = Item
-
-  static propTypes = {
-    borderSide: PropTypes.oneOf(['left', 'right']),
-  }
-
-  static defaultProps = {
-    borderSide: 'left',
-  }
-
-  constructor(props) {
-    super(props)
-
-    this.state = {
-      borderSide: props.borderSide,
-    }
-  }
-
-  componentDidUpdate(prevProps) {
-    if (prevProps.borderSide !== this.props.borderSide) {
-      this.setState({ borderSide: this.props.borderSide })
-    }
-  }
-
-  render() {
-    const { children } = this.props
+export const HeaderMenu = ({
+  backgroundColors,
+  borderSide,
+  borderSideOnHover,
+  largeItem,
+  noBorder,
+  children,
+}) => (
+  <Context.Provider
+    value={{
+      backgroundColors: backgroundColors,
+      borderSide: borderSide,
+      borderSideOnHover: borderSideOnHover,
+      largeItem: largeItem,
+      noBorder: noBorder,
+    }}
+  >
+    <nav role="navigation">
+      <List noBorder={noBorder} role="menubar">
```

Comment:
```suggestion
      <List noBorder={noBorder}>
```

https://adrianroselli.com/2017/10/dont-use-aria-menu-roles-for-site-nav.html